macros/collect_freshness.sql

```diff
@@ -8,7 +8,7 @@
 
   {%- set enabled_array = [] -%}
   {% for node in graph.sources.values() %}
-    {% if node.name == source.name %}
+    {% if node.identifier == source.identifier %}
```

Comment:
if an identifier or alias isn't explicitly provided to a source, is `identifier` the `name` by default? @fivetran-joemarkiewicz

Reply:
That is what I got from my testing. When using this on nodes with **only** the `name` config, it just defaulted to use the name.

Reply:
ah nice seems like it does